src/Library/demos/Launcher/main.js

```diff
@@ -20,59 +12,56 @@ Gio._promisify(
 Gio._promisify(Gtk.FileDialog.prototype, "open", "open_finish");
 Gio._promisify(Gtk.UriLauncher.prototype, "launch", "launch_finish");
 
-const file_launcher = new Gtk.FileLauncher();
-const uri_launcher = new Gtk.UriLauncher();
-const file = Gio.File.new_for_path(pkg.pkgdatadir).resolve_relative_path(
-  "Library/demos/Launcher/workbench.txt",
-);
-file_launcher.set_file(file);
-file_launcher.always_ask = true;
```

Comment:
we can set properties directly when instanciating see below `new Gtk.FileLauncher({`

Reply:
Also added to Style Guide for future reference :)